nmdc_server/cli.py

```diff
@@ -177,3 +180,91 @@ def shell(print_sql: bool, script: Optional[Path]):
         c.InteractiveShellApp.file_to_run = script
 
     start_ipython(argv=[], config=c)
+
+
+@cli.command()
+@click.option("-u", "--user", help="NERSC username", default=os.getenv("USER"), show_default=True)
+@click.option("-h", "--host", help="NERSC host", default="dtn01.nersc.gov", show_default=True)
+@click.option("--list-backups", is_flag=True, help="Only list available backup filenames")
+@click.option(
+    "-f",
+    "--backup-file",
+    help=(
+        "Filename in NERSC's backup directory to load. "
+        "If not provided, the latest backup will be loaded."
+    ),
+)
+@click.option(
+    "-k",
+    "--key-file",
+    default="/tmp/nersc",
+    show_default=True,
+    help="Path to NERSC SSH key file within Docker container. Use if not using standard mounting.",
+)
+def load_db(key_file, user, host, list_backups, backup_file):
+    """Load a local database from a production backup on NERSC."""
+    pgdump_dir = "/global/cfs/cdirs/m3408/pgdump"
+
+    if list_backups or not backup_file:
+        click.echo("Finding latest production backups...")
+        proc = subprocess.run(
+            [
+                "ssh",
+                "-i",
+                key_file,
+                "-o",
+                "StrictHostKeyChecking=no",
+                f"{user}@{host}",
+                f'find {pgdump_dir} -type f -iname "*pg_main-prod*.dump" -printf "%f\n"',
+            ],
+            capture_output=True,
+            encoding="utf-8",
+        )
+        if proc.returncode != 0:
+            click.echo("Error finding backups:")
+            print(proc.stderr)
+            sys.exit(1)
+
+        dumps = sorted(proc.stdout.splitlines())
+        if list_backups:
+            click.echo("Available backups:")
+            for dump in dumps:
+                click.echo(dump)
+            return
+
+        if not dumps:
+            click.echo("No backups found")
+            sys.exit(1)
+
+        backup_file = dumps[-1]
+
+    click.echo(f"Downloading {backup_file}...")
+    subprocess.run(
+        ["scp", "-i", key_file, f"{user}@{host}:{pgdump_dir}/{backup_file}", "."],
+        check=True,
+        encoding="utf-8",
+    )
```

Comment:
Do you think its worth checking to see if the most recent backup already exists in the container first?